scripts/scil_summarize_bundles_similarity_measures.py

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+import copy
+import hashlib
+import itertools
+import json
+import multiprocessing
+import os
+import shutil
+
+from dipy.io.stateful_tractogram import Space, StatefulTractogram
+from dipy.io.streamline import load_tractogram, save_tractogram
+from dipy.io.utils import is_header_compatible, get_reference_info
+from dipy.segment.clustering import qbx_and_merge
+import nibabel as nib
+import numpy as np
+from numpy.random import RandomState
+
+from scilpy.io.utils import (add_overwrite_arg,
+                             add_reference,
+                             assert_inputs_exist,
+                             assert_outputs_exist,
+                             link_bundles_and_references)
```

Comment:
we should remove the **s** in reference since you link multiple bundles with one reference